example/src/Examples/NativeBottomTabs.tsx

```diff
@@ -9,7 +9,7 @@ const Tab = createNativeBottomTabNavigator();
 
 function NativeBottomTabs() {
   return (
-    <Tab.Navigator>
+    <Tab.Navigator inactiveTintColor="blue">
```

Comment:
For JS Bottom tabs we have 2 props: `tabBarActiveTintColor`, `tabBarInActiveTintColor` to provide this functionality, which are part of screenOptions.

To make API close to JS one don't you think we should include this inside screenOptions and match names as well?